bahmni-docker/docker-compose.yml

```diff
@@ -212,6 +212,12 @@ services:
     environment:
       - OPENMRS_HOST=${OPENMRS_HOST:?}
 
+  appointments:
```

Comment:
Also we would need to add configuration for local development setup using volume mount. Please refer bahmni-web service.